### DISCUSSION TITLE
Django Admin Site Support

### DISCUSSION THREAD

**Discussion by saanpritom** [django-tenants/django-tenants#699] [Q&A]
Currently I can't find any support or documentation for the Django Admin Panel for private tenants. Is there any solution of this?

**Comment by tomturner** [comment]
It works automatically. Just use it like the normal admin

**Reply by saanpritom** [reply]
Yeah but it is saying, Schema not allowed. Do I need to do some permission stuffs on the database side?

**Reply by mast3rbow** [reply]
Do you have django.contrib.admin in your Tenant_apps in settings.py? 

If not it from my experience and @tomturner can correct me, but it won't provide admins for tenants, So i have in both public and tenants.

**Comment by SilverFoxA** [comment]
@saanpritom This is not a problem, it works out of the box if you have your Django project setup correctly. Check your URLs file and see if the admin links are added.
